src/core/agent/services/identifierService.ts

```diff
@@ -230,6 +230,9 @@ class IdentifierService extends AgentService {
           pendingDeletion: false,
         }
       );
+      await this.props.signifyClient.identifiers().update(localMember.id, {
+        name: `XX-${randomSalt()}:${localMember.displayName}`,
```

Comment:
The local member has the groupId in the name. Can we preserve that for historical reasons?

Reply:
So I’ll format the name for this case as:
`name: XX-${randomSalt()}:${localMember.groupMetadata?.groupId}:${localMember.displayName}`
Is that okay?